test/Microsoft.AspNet.Mvc.Formatters.Json.Test/JsonInputFormatterTest.cs

```diff
@@ -187,6 +195,13 @@ public static IEnumerable<object[]> JsonFormatterReadSimpleTypesData
             Assert.True(result.HasError);
             Assert.Equal("The supplied value is invalid for Byte.", modelState["[2]"].Errors[0].ErrorMessage);
             Assert.Null(modelState["[2]"].Errors[0].Exception);
+            Mock.Get<ILogger>(logger).Verify(l => l.Log(
+                LogLevel.Verbose,
+                It.IsAny<int>(),
+                It.IsAny<object>(),
+                It.IsAny<Exception>(),
+                It.IsAny<Func<object, Exception, string>>()),
+            Times.Once);
```

Comment:
That when we have an error coming out of the formatter we log the error in addition to returning it. I suppose it's probably mostly paranoia on my to double check that the logging happened, I I'll remove it.